src/snowflake/snowpark/_internal/analyzer/schema_utils.py

```diff
@@ -90,7 +91,20 @@ def analyze_attributes(
             session._conn._cursor.description, session._conn.max_string_size
         )
 
-    return session._get_result_attributes(sql)
+    # collect describe query details for telemetry
+    stack = traceback.extract_stack(limit=10)[:-1]
+    if len(stack) >= 2:
+        stack_trace = [frame.line for frame in stack]
+    else:
+        stack_trace = None
```

Comment:
are we doing this to figure out which is the biggest contributor of describe queries?

Reply:
yes